equinix/resource_metal_device_acc_test.go

```diff
@@ -905,7 +905,7 @@ resource "equinix_metal_device" "test" {
 	operating_system = local.os
 	billing_cycle    = "hourly"
 	project_id       = equinix_metal_project.test.id
-	user_ssh_key_ids = [equinix_metal_ssh_key.test.id]
+	user_ssh_key_ids = [equinix_metal_ssh_key.test.owner_id]
```

Comment:
This fixes the 422 error from the API, but the test is currently checking that the device has 2 SSH keys, and that is an invalid expectation.  It has every SSH key for the users specified in `user_ssh_key_ids` plus the 1 project SSH key specified in `project_ssh_key_ids`.  I'm not really sure how to resolve this in a reliable way; can we instead check that the device SSH keys include the project and user SSH keys that were created as part of the test?